build-scripts/build-custom-app.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# Define the content for the main.tsx file
+read -r -d '' MAIN_CONTENT <<EOM
+
+import React, { FC, ReactNode } from "react";
+import Header from "@components/headers/Main";
+import Footer from "@components/footer/Footer";
+
+type TProps = {
+  children?: ReactNode;
+};
+
+const Main: FC<TProps> = ({ children }) => (
+  <>
+    <Header />
+    <main id="main" className="flex flex-col flex-1">
+      {children}
+    </main>
+    <Footer />
+  </>
+);
+export default Main;
+
+EOM
+
+# Define the content for Analytics.tsx
+read -r -d '' ANALYTICS_CONTENT <<EOM
+
+import React from 'react';
+
+const Analytics = ({enableAnalytics}: {enableAnalytics: boolean}) => {
+    return null 
+};
+
+export default Analytics;
+
+EOM
+
+# Define the content for methodology.tsx
+read -r -d '' METHODOLOGY_PAGE_CONTENT <<EOM
+
+const Methodology = () => {
+  return null;
+};
+
+export default Methodology;
+
+EOM
+
+# Define the content for homepage.tsx
+read -r -d '' HOMEPAGE_CONTENT <<EOM
+
+const LandingPage = ({}) => <h1> Hello World </h1>;
```

Comment:
I think it would be good to include the search bar on the skeleton landing page with the theme name/customer name, as although the search isn't a required file needed to make the skeleton work, it will always be included in every custom app we do :)